tools/cli/commands.go

```diff
@@ -613,22 +620,27 @@ func RunWorkflow(c *cli.Context) {
 	tcCtx, cancel := newContextForLongPoll(contextTimeout)
 	defer cancel()
 
-	resp, err := serviceClient.StartWorkflowExecution(tcCtx, &s.StartWorkflowExecutionRequest{
+	startRequest := &shared.StartWorkflowExecutionRequest{
 		RequestId:  common.StringPtr(uuid.New()),
 		Domain:     common.StringPtr(domain),
 		WorkflowId: common.StringPtr(wid),
-		WorkflowType: &s.WorkflowType{
+		WorkflowType: &shared.WorkflowType{
 			Name: common.StringPtr(workflowType),
 		},
-		TaskList: &s.TaskList{
+		TaskList: &shared.TaskList{
 			Name: common.StringPtr(taskList),
 		},
 		Input:                               []byte(input),
 		ExecutionStartToCloseTimeoutSeconds: common.Int32Ptr(int32(et)),
 		TaskStartToCloseTimeoutSeconds:      common.Int32Ptr(int32(dt)),
 		Identity:                            common.StringPtr(getCliIdentity()),
 		WorkflowIdReusePolicy:               reusePolicy,
-	})
+	}
+	if c.IsSet(FlagCronSchedule) {
```

Comment:
You also need to add this flag to to Run command, like the one you already added for Start command.